test/fuzz/test_runner.py

```diff
@@ -14,6 +14,7 @@
 
 # Fuzzers known to lack a seed corpus in https://github.com/bitcoin-core/qa-assets/tree/master/fuzz_seed_corpus
 FUZZERS_MISSING_CORPORA = [
```

Comment:
on a second thought, this is a bit too strict and causes many merge conflicts. I think we can remove `FUZZERS_MISSING_CORPORA` and create the dir unconditionally. With a comment it should be fine. I believe this was your initial suggestion on how to handle the two repos going out of sync.

Reply:
`FUZZERS_MISSING_CORPORA` now removed. Now informing the user about missing seed corpora with a call to action :)